extra/html-extra/HtmlExtension.php

```diff
@@ -102,10 +103,10 @@ public static function htmlClasses(...$args): string
                     if (!$condition) {
                         continue;
                     }
-                    $classes[] = $class;
+                    $classes[] = (string) $class;
                 }
             } else {
-                throw new RuntimeError(\sprintf('The "html_classes" function argument %d should be either a string or an array, got "%s".', $i, get_debug_type($arg)));
+                throw new RuntimeError(\sprintf('The "html_classes" function argument %d should be either a string, an instance of %s or an array, got "%s".', $i, Markup::class, get_debug_type($arg)));
```

Comment:
I would not change the exception message as the Markup class is more an implementation detail than something people needs to be aware.

Reply:
Done!